niworkflows/data/getters.py

```diff
@@ -22,7 +22,8 @@
     'mni152_nlin_sym_las': ('57fa7fc89ad5a101e635eeef', '9c4c0cad2a2e99d6799f01abf4107f5a'),
     'mni152_nlin_sym_ras': ('57fa7fd09ad5a101df35eed0', '65d64ad5a980da86e7d07d95b3ed2ccb'),
     'mni_icbm152_linear': ('580705eb594d9001ed622649', '72be639e92532def7caad75cb4058e83'),
-    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec')
+    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec'),
+    'oasis_dtk31_mni152': ('5a95995e91b689000ea221fe', '6eb144c5e3f5a375f23ffc3dc2d1cc2f')
```

Comment:
Can you add a comma at the end here, so that future additions only affect the one line?